src/main/java/chess/domain/position/ChessFile.java

```diff
@@ -0,0 +1,64 @@
+package chess.domain.position;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public enum ChessFile {
+    A("a", 0),
+    B("b", 1),
+    C("c", 2),
+    D("d", 3),
+    E("e", 4),
+    F("f", 5),
+    G("g", 6),
+    H("h", 7);
+
```

Comment:
```suggestion
    G("g", 6),
    H("h", 7),
    ;

```

다음 enum이 추가될 때를 생각해서 위처럼 쓰기도 합니다!

Reply:
미션 상에서는 추가되지 않겠지만
만약 실제 프로덕트의 코드였다면 확실히 다음 enum이 추가되는 경우 불편함이 덜겠군요!
하나 배워갑니다! 적용해보겠습니다:)